tests/func/test_100_register.py

```diff
@@ -72,3 +73,69 @@ def test_cancelled(controller, reader_driver):
             input_field = controller.driver.find_element_by_css_selector(
                 '#side-panel-adder > #bookmark-name')
             assert input_field.text == ""
+
+    @staticmethod
+    def test_eexist(controller, reader_driver):
```

Comment:
"eexist"?

Reply:
meaning "error because it already exists" (following ERRNO's naming which seemed like a common knowledge to me), as the point of this test is to validate this failure behavior of the registration.